build_depends.repos

```diff
@@ -28,7 +28,7 @@ repositories:
   universe/external/grid_map:
     type: git
     url: https://github.com/ANYbotics/grid_map.git
-    version: ba2f9cb6e62f7ee9c5bac7401391a211e442e459
+    version: 74333f037cfce321248dfa7b954a815d4f67d79d
```

Comment:
I was going to say "Did we not remove grid_map?"

But then found https://github.com/autowarefoundation/autoware.universe/pull/2429

Reply:
we made a comment on the exact same thing.
But you were one step ahead of me to notice the draft PR